XRTK.SDK/Packages/com.xrtk.sdk/Runtime/Features/TeleportSystem/BaseTeleportHandler.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using XRTK.EventDatum.Teleport;
+using XRTK.Interfaces.TeleportSystem;
+using XRTK.Interfaces.TeleportSystem.Handlers;
+using XRTK.Services;
+
+namespace XRTK.SDK.TeleportSystem
+{
+    /// <summary>
+    /// Base implementation for handling <see cref="IMixedRealityTeleportSystem"/> events in a
+    /// <see cref="MonoBehaviour"/> component.
+    /// </summary>
+    public abstract class BaseTeleportHandler : MonoBehaviour, IMixedRealityTeleportComponentHandler
```

Comment:
Should the Base TeleportHandler not be part of Core?

Reply:
I agree but I am unsure how to proceed. Didn't we agree to not put MonoBehaviours in Core? Also the BaseHandler is abstract and cannot be used on its own. I guess we could make it "instant teleport" by default or something. But I'd rather move the default implementation to Core then as well. I want to keep the Base abstract really. It's the starting point for teleportation implementations and should really be just the bare bones to recieve the proper events. Just as the "Global Input Handler" basically.

Reply:
I do belive it belongs to SDK. If people want out of the box teleporting they need to get SDK. If they want to go full "custom" and not use SDK they will have to implement many things anyways. I think the Input System should really work on its own, even without SDK. But for all other systems, SDK it is.

Will will run into this issue many times from now on I think and we need to define the way forward.

Reply:
I agree, if it requires a MonoBehaviour it should live in the SDK

Reply:
Well we bend the rules here for this with the default gaze provider and other solvers/utilities

Reply:
But for now I'm fine with it the way it is

Reply:
After reviewing this and playing with it a bit more I think that the handlers could have been data providers. I'm not sure if we're explicitly required to use a monobehaviour for these except for when we're handling the teleport request events themselves, but then again we could just make those basic c# events.